app/javascript/suggestions.js

```diff
@@ -37,7 +34,6 @@ Input.prototype.init = function () {
     this.$formGroup.appendChild(this.$suggestionsHeader)
     this.$formGroup.appendChild(this.$ul)
 
-    this.$module.removeAttribute('list') // [change: deactivate browser-native suggestions]
```

Comment:
must admit hadn't spotted that one - this change is harmless as we don't support them, but should we? Presumably there are preventDefault() calls that would make this safe if we enabled the browser version of this?

Reply:
I'm second guessing my rationale on this: 

- Based on my very basic understanding of how the [datalist](https://www.marioaraque.com/native-html5-autocomplete) element works, you would add the `list` attribute to the `input` field to get the browser-native suggestions.
- Because we don't add the `list` attribute to the `input` field, it didn't make sense to have `removeAttribute('list')`

I hadn't delved into the if we need to support them, in the first place. Please do correct me if I've mis-understood.

Reply:
I think it might be one for Kristina

Reply:
If that's the case, I start a slack conversation, to explore the the possibility of using the browser version's of the autocomplete/suggestion list? 

For the moment I'll add this bit back in so we can get the other bits merged.